db/migrate/20171026141412_add_user_stamps_to_course_video_topics.rb

```diff
@@ -0,0 +1,38 @@
+class AddUserStampsToCourseVideoTopics < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :course_video_topics,
+                  :creator,
+                  references: :users,
+                  foreign_key: { to_table: :users }
+    add_reference :course_video_topics,
+                  :updater,
+                  references: :users,
+                  foreign_key: { to_table: :users }
+
+    Course::Video::Topic.reset_column_information
+    infer_topic_creator
+
+    change_column :course_video_topics, :creator_id, :integer, null: false
+    change_column :course_video_topics, :updater_id, :integer, null: false
+  end
+
+  def infer_topic_creator
+    ActsAsTenant.without_tenant do
+      Course::Video::Topic.all.each do |topic|
```

Comment:
Just pointing out, normally we use `find_each` to loop through all records 😅  But since there are no much video topics in the system, can leave as it is.